src/TouristInfoCards.js

```diff
@@ -0,0 +1,59 @@
5Nze+ft0W3gnxHTaxrXzAHQXD6rjlpPp7pK1WrTuoRTsW0qdGok5ys2cq+k4QBT75bJBkZ/Yhd9w74domky5ubWnbu0E/xdyUHdxaiTLmjEkFog9AMgjrO6mDRq+PmEZIAJpg4P9QlYtTUq1UlZxa+DVp6VOm201IAVtS0Xm9ziIJAAbv8A1OuPUqFbVgX+GbbXC5zzuAdgQ3qhtWu2XyR4mt2zkR2UH6oEmATLWj2j9F1HNs5GEUEKuteOZGP+WRaGt338QE/iqOJ1PqMzhp93LE+s4ziJAHc4jop1GvcDc0wQB0GxlL2PdBV9YYJ7BZKvEAGwMysNRwG7hO2M7dO0qg1f5R9zlM6nAEka3alxzsPZZzUE/wAx/BW6HROrOMugDc7x9pH9h5hGdFoKIe2m0B7nHDnyGdcFv23z07pMm/I+ABNxMT9ht/ldLwPhlV7LnW8szF03YJBtA322/FHKnAaVKa1YGpGAGAcvqQAGnPXc9fKBg4hxMCCXEAQA0RdAiAA0CB+5TU4uXN+PkWo1Dh9/BgfoqbahBLWtGQ7xEEdrTt67KzVaqgHDlM+kACCeg6uJPXOEPdrHVAGwA0dBufU/2CkT1/t2WmMV4Mk5vyXvrFxuMSewhPTfCou/eU4cP3Kvy8GZR5uWud6q6i8+f7+4WUn09j/cqTft7IN3VgqFpXDOkjr+MfqVncwBxKhpqkdkz35WJ0/7rkdVTvRSLw5QvMqs1VEVPRXcme6YT0bo3/KP8fvqs1OpE+qenUhvUemR/j0WKvUz6qihTe5Jl+pq/wBuK+DTV1KWm8bvIBxPtGPdYC5OH7+y2VI+xqPZzoVL1E5dBWtxCvUqPa6qWgOIaJtx9v3hZq9M0w5wggiHeKceqx3Ihwymx9zamdoh1p/yseppQpUc2uvodDTamVWtivP1Lfm+WWtk2uptI6Ay0GMf1Sfuh2q1sspkhpHiwSTnEHEHut2s0lFrS4ve0Af0nHYRHbsglPSOq1WsYx9rgSMeIgA5j7fmuZB0pvJfsdSU6kY4/Jt4bxgghjnQ3MQfpwYEkyM5nyRvWfFjHRSNMVKeHZJnYS4Hznby6LneK8EDA5zHgibQ1wIeDuS72PuCsGmvpvZNpEg74z0P76q9UoySlFGd15xeMmdXV0On1Ia8sr0seENh4JMGWhxutgdxgSN1yNdlrnC5zgDg5bMdSD+S6elw7VFwrUOY9hNrbqgL2wDjLpxaYOJhaj8P1dRSLjZDAXSIvkzJDQMzBkSr6TS7lz/ooq3d7R/6c0ymHNbeB6Sf7R+qk2g0CAB5dfxRLg/AxUJbU1PLIyAWxcTMC4nHQLTwbggqmo11S004mesz1PTG/muhuwTd1yjkuhUlxB8N9ASpUxmntu4Hxfl+v2VfIp1CBSdBnNxIcY7tE2/vdHNVwirTaXBpc0nBGSQLswPT8kDqacQY6mexmMGUk4+Yq6L4TtxJ2YztPWbGzp2HX8gQqauptJaQQR2II77raOH6pzbiZuIHizjvkYwiVPg0iXET5Mn8SVllX0/Tlb8zXHTapq+N/wAjmqtEA3PeyT0PT26qsaim0QA5/rhvtssQp91MUghufCFUPllh1h/gAbvsM58zkKp5c7JJPqZVoYntStyfY9kuiptJWtYpAKTAokBss05c0y1xG/nv2nb1C06bWVWPDw4kjIk/3VVOm4gkAkDcgEgepTK1QTE35QaaClbjVZ4NxaPQEme4JOPZD7Ou5O5JyVEPTgq2MV0Uym23IsH291Ien4quQpBWLgqtcmHevupA+ZUAnlG4MSyU7SqwU8o3CkaWVIwmNRZ7ki5L5HcuLFxqJXqm5KU3AlzYyrjEjzEwfVVuM5VDXqQd0UXBJPJWJXJpUSmlPkU4lkqLxKaU8oSSkrMkG4STRVUouaLwZthwHmDO32R3h/GKd7K3Mi0GaThBMtLTbUjPTfJ6lA6mqmWjYb/osRoz9K5VXSxkdmlqXGx1fxFUbVbSrU//ADAZ7gtjfpImPZc68AiDn1hXijVpNFN+24Ak2k4g9jj8FRQ0xzcOp7YV+lpSjBU1yijV1YObm+zqPh7iL20m2vAdBAccgG+QSOo29/NSpccqsqvbWeC0kgWgMaRADiDE3SBgrnyLRAJAHaQn0TmuPKqfS8wHblpOzvtOe4lNqNOl7muQaXUtuyfHg6z4g4bSq06dzxzHAkEEwc/SMQSuf02uq6R9pYYDQDjxFgI6PydhtEQuu4LoGUmMDKjqhaQ4uhzpDpgMBMNwMwei08Ucy26q2Q5wMVC0R4gAAM+v/dcZf1RJ7cY5Lr6nVl/T7vccsX9jGPiKgAAXl4cG+Jog7dRPhMz5LnuLUGEm0GSARADvUEtMSB12ONkB4dUDnPaCB43ESJkF0ANIyNwjbeHPawvkA/y7u/D8l6DSKKh2cDWym5dETxwWQB4mgDyk49v1WVvGnxmPY/qgnEH+MxiZmJE5IEifRbdDqKTWAOoB56uLv8LLKhSu/bc3R1NaS/HiCw1PCSSQVjwnASSTpC3HISCdJFCtl1PUOALQYBmcDqIOemMYVaSSdJJ3Qjd1ySaFMNTpJrgUUOFIOSSTC+RSnlJJG4bCDki5JJS4LClK5JJEDQ1ye5OkiKIFTLtkklFyFqw1yaU6ScRiBSfsfRJJR9MEVyZNK3fzVmjZDh2kH2KSSy29qNbfvYe4pUcahY47GzaBguIO+dz7rLQoudUFICS/DJ8OfU46JJKyEnDSKS7sU1IKWrcX1ct4loX0Khp1LQR2yDgHBx3Co4gxtJzLXXBwkGBIIIke3Xokkjm50FJ92RFFQrOK6Cul+JqgaQ653hAEkNiPpIjp3HXCFavitzgXRO8GXR4pgE9EySxx01OnLKK5ZunqKkoqLfAD4bUsqNJ7ie+CD/ZHnaokeFx2OxnJdJ/RJJbNPymjDquHcu1DWGi+sDSvAJIIteSBDfCCQ7MZQb5Vpgl5mBuP8JJLM6soSl5/n0NLpqaj4/n1P//Z"
```

Comment:
For the future - could you find a URL of the image online and then put that in the `src` it makes it quite a bit easier to read

Reply:
Correction done
Thanks Chris